bin/test_commit

```diff
@@ -0,0 +1 @@
+bin/test && bin/vet && bin/lint_git
```

Comment:
What is purpose of `bin/test_commit` helper?
Example usage documentation missing.

Reply:
@Waldz added docs.